node_modules/oae-core/documentpreview/documentpreview.html

```diff
@@ -52,10 +52,13 @@
 
 <!-- TEMPLATES -->
 <div id="documentpreview-content-page-template"><!--
-    <div class="documentpreview-content-page ${cssScopeClass}" data-page-number="${pageNumber}">
-        <div class="documentpreview-content-page-loading text-center">
-            <i class="fa fa-spinner fa-spin"><span class="sr-only">__MSG__LOADING__</span></i>
+    <div class="documentpreview-content-page" data-page-number="${pageNumber}" >
+        <div class="documentpreview-content-html-page ${cssScopeClass}" aria-hidden="true">
+            <div class="documentpreview-content-page-loading text-center">
+                <i class="fa fa-spinner fa-spin"><span class="sr-only">__MSG__LOADING__</span></i>
```

Comment:
I guess this means that the loading text will no longer be read as it is in an `aria-hidden` block? Would it be worth pulling the loading icon out of the `documentpreview-content-html-page` block?